account_invoice_blocking/models/account_invoice.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2016 Acsone SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountInvoice(models.Model):
+
+    _inherit = 'account.invoice'
+
+    @api.multi
+    def _get_move_line(self):
+        """
+        This method searches for payable or receivable move line
+        of the invoice
+        :returns payable or receivable move line of the invoice
+        """
+        self.ensure_one()
+        type_receivable = self.env.ref('account.data_account_type_receivable')
+        type_payable = self.env.ref('account.data_account_type_payable')
+        user_type_id_list = [type_receivable.id, type_payable.id]
+        return self.env['account.move.line'].search([
```

Comment:
@astirpe Can we check also that the move account is equal to the invoice account, we have advance invoices where we also have a payable/receivable on the invoice line, so i want to trace that one

Reply:
@feketemihai I'm quite busy this period, I'm afraid I will not able to look at it during the next weeks. If you already have implemented a working solution, please open a PR to this branch and I will merge it asap. Thank you!